src/Microsoft.Azure.WebJobs/TableAttribute.cs

```diff
@@ -40,6 +41,11 @@ public class TableAttribute : Attribute, IConnectionProvider
         private readonly string _partitionKey;
         private readonly string _rowKey;
 
+        // Table Name Rules are summerized in this page https://msdn.microsoft.com/en-us/library/azure/dd179338.aspx
+        // Table names may contain only alphanumeric characters, can not start with a numeric character and must be 
+        // 3 to 63 characters long
+        private const string TableNameRegex = "^[A-Za-z][A-Za-z0-9]{2,62}$";
```

Comment:
I'll comment abut that on the new issue 479. 

---
In reply to: [123435508](https://github.com/Azure/azure-webjobs-sdk/pull/1188#discussion_r123435508) [](ancestors = 123435508)